Doc/library/dataclasses.rst

```diff
@@ -97,7 +97,24 @@ Module contents
      the name and repr of each field, in the order they are defined in
      the class.  Fields that are marked as being excluded from the repr
      are not included.  For example:
-     ``InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)``.
+     .. doctest::
+
+        >>> from dataclasses import dataclass, field
+        >>> @dataclass(repr=False)
+        ... class InventoryItem:
+        ...     name: str
+        ...     unit_price: float
+        ...     quantity_on_hand: int
+        >>> InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)  # doctest: +ELLIPSIS
+        <__main__.InventoryItem at 0x...>
+
+        >>> @dataclass
+        ... class InventoryItem:
+        ...     name: str
+        ...     unit_price: float
+        ...     quantity_on_hand: int = field(repr=False)
+        >>> InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)
+        'InventoryItem(name='widget', unit_price=3.0)'
```

Comment:
This is still not the correct place. Please put it in the **other** section (below) which is about `field()`. This paragraph is *only* about the `@dataclass` decorator.

In addition I don't think that there should be a `'...'` in the output.